test/e2e/specs/specs-playwright/wp-woocommerce__landing-page.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	DataHelper,
+	SidebarComponent,
+	TestAccount,
+	WoocommerceLandingPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'WooCommerce Landing Page' ), function () {
+	let page: Page;
+
+	describe( 'Learn more button opens a support article', function () {
+		let landingPage: WoocommerceLandingPage;
+
+		beforeAll( async () => {
+			page = await browser.newPage();
+
+			const testAccount = new TestAccount( 'defaultUser' );
+			await testAccount.authenticate( page );
+		} );
```

Comment:
This design of having two distinct, small describe blocks, each with their own new page, is totally valid if that's how you stylistically prefer to do the tests!

As an alternative, it would also be just fine to merge something like this into one single flow (where for example, you navigate to the woo page, open the support dialog, close it, and then launch the installer). We often design specs like this, where we view our test design from the perspective of "what path would a user go through" as opposed to making a bunch of small, atomic tests. But, either approach is valid, and I will defer to you! 🙂

Reply:
Prefer tests to assert single things where possible. Is there much of a hit to total run time to writing tests this way? That's probably the thing that would sway me the other way. Will leave as is for now.

Reply:
Changed my mind on this one, looks better as a single unit.